tests/gui/test_formbuilder.py

```diff
@@ -132,3 +140,93 @@ def test_string_list_widget(qtbot):
 
     widget.setValue(["zip", "cab"])
     assert widget.text() == "zip cab"
+
+
+def test_exponential_spin_box(qtbot):
+    """I am testing the exponential spin box class itself not with the gui"""
```

Comment:
We aren't concerned with what YOU are doing, but rather what the function is doing:
```suggestion
    """Test ExponentialSpinBox setValue and stepBy methods."""
```

> The docstring may be descriptive-style ("""Fetches rows from a Bigtable.""") or imperative-style ("""Fetch rows from a Bigtable."""), but the style should be consistent within a file.

Ref: [google-style docstrings](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)